src/Subscriber/WelcomeFirstTimeContributorSubscriber.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Subscriber;
+
+use App\Event\GitHubEvent;
+use App\GitHubEvents;
+use App\Issues\CommentsApiInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class WelcomeFirstTimeContributorSubscriber implements EventSubscriberInterface
+{
+    private $commentsApi;
+
+    public function __construct(CommentsApiInterface $commentsApi)
+    {
+        $this->commentsApi = $commentsApi;
+    }
+
+    public function onPullRequest(GitHubEvent $event)
+    {
+        $data = $event->getData();
+        if ('opened' !== $data['action']) {
+            return;
+        }
+
+        if ('NONE' !== ($data['pull_request']['author_association'] ?? '')) {
```

Comment:
According to https://docs.github.com/en/free-pro-team@latest/graphql/reference/enums#commentauthorassociation , `FIRST_TIMER` or `FIRST_TIME_CONTRIBUTOR` may also be correct values for the association to show this welcome message?